src/main/java/com/minecolonies/items/AbstractItemMinecolonies.java

```diff
@@ -26,5 +30,7 @@ public AbstractItemMinecolonies()
      *
      * @return      Name of the item
      */
-    public abstract String getName();
+    public String getName() {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this left curly brace to the beginning of next line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)